docs/guides/networking/linode-network/ip-failover/index.md

```diff
@@ -137,6 +133,17 @@ iface lo inet static
 
         If you receive the following output, you can safely ignore it: *RTNETLINK answers: Cannot assign requested address*.
 
+    -   **CentOS/RHEL**: Using [NetworkManager](https://en.wikipedia.org/wiki/NetworkManager). Since NetworkManager does not support managing the loopback interface, you need to first add a dummy interface named *shared* (or any other name that you wish). Instead of editing the file directly, the [nmcli]() tool is used.
```

Comment:
@wildmanonline [nmcli]() is an empty link, not sure where the appropriate target URL was for it

Reply:
Good catch. Adding in a link to the man pages now. I didn't have a specific link in mind when I put the placeholder in, but figured it would be beneficial to link to more information about that command in case the user needs additional help.